src/main/java/com/kodilla/ecommercee/controller/OrderController.java

```diff
@@ -0,0 +1,40 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.dto.OrderDto;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@CrossOrigin("*")
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    @GetMapping
+    public ResponseEntity<List<OrderDto>> getOrders() {
+        return ResponseEntity.ok(new ArrayList<>());
+    }
+
+    @PostMapping
+    public ResponseEntity<Void> addOrder(@RequestBody OrderDto orderDto) {
+        return ResponseEntity.ok().build();
+    }
+
+    @GetMapping("/{orderId}")
+    public ResponseEntity<OrderDto> getOrder(@PathVariable Long orderId) {
+        return ResponseEntity.ok(new OrderDto(1L, 1L, 1L));
```

Comment:
Okey, I can change that but is it necessary? becouse its part of the service to menage Logic which we dont have yet, so do we really need to test Logic in our Controller if it wasnt our task?

Reply:
The question is what is the purpose of this task. If we have to prepare tests based on it next (in another task), we are not able to do it in reasonably way because they will never return an error (so what would they be used for if they don't simulate real situation?). This code is formally ok (as I wrote in the main comment) but in this case, it is only art for art's sake because it will require a lot of reconstruction to go further with the next tasks...
but I emphasize that this is only my, personal, opinion, my observation...